src/screens/Events.tsx

```diff
@@ -98,7 +106,12 @@ class Events extends React.Component<Props, {}> {
           </Block>
           <Block flex={1}>{this.renderChart()}</Block>
         </Block>
+        <Button
```

Comment:
Why do we need a button for loading data? Shouldn't this happen without the user knowing or by using the  GET_EVENTS action?

Reply:
No this one is for like repopulating data on the realm, its just temporary I needed a way to interact with the realm actively instead of passively so I can control tests